src/openms/source/ANALYSIS/ID/IDScoreGetterSetter.cpp

```diff
@@ -83,6 +136,12 @@ namespace OpenMS
       {
         auto [isDecoy, tgt_accession] = removeDecoyStringIfPresent_(acc, decoy_string, decoy_prefix);
         const double tgt_proportion = picked_scores.at(tgt_accession).second;
+        // The problem is here, that in theory, the matching pair can be in different groups
```

Comment:
not sure if I understand this comment

Reply:
Because it is not an easy problem. But I can't write an article with pictures inside here ;)